pkg/app/gen/html.go

```diff
@@ -1284,6 +1284,11 @@ var attrs = map[string]attr{
 	},
 
 	// K:
+	"key-value": {
+		Name: "SetAttribute",
+		Type: "key|value",
+		Doc:  "Corresponding to Javascript Dom operations setAttribute",
```

Comment:
This should say what it does rather than stating it is equivalent to something. 
=> `sets the named attribute with the given value.`